theme/n21/views/lib/footer.hbs

```diff
@@ -1,7 +1,7 @@
 <footer class="hide-inline small-footer">
 	<div class="imprint">
 		<div class="center">
-			<a>&copy; 2018 </a>
+			<a>&copy; {{currentYear}} </a>
```

Comment:
is the theme short title removed here intentionally?

Reply:
I don't know, but that's not part of this ticket.